### PR TITLE
Change photo from emotions picture to ours please merge fast and delploy

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -100,14 +100,19 @@
...
+            <div class="container" style="height:450px">
+            </div>
+        </div>
+    </section>
+
+    <section id="helfer" class="container content-section text-center">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2">
+                <h2>Helfer</h2>
+                <p>Auch du kannst mit helfen besuche dazu einfach unser Helfersystem.</p>
```
Comment (SlothOfAnarchy) [inline index.html]: *mithelfen

File: index.html [hunk 2]
```diff
@@ -100,14 +100,19 @@
     </section>
 
     <!-- Download Section -->
-    <section id="download" class="content-section text-center">
+    <section id="download" class="content-section text-center" >
```
Comment (SlothOfAnarchy) [inline index.html:103]: download, huh.

**Review by SlothOfAnarchy — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (Scriptkiddi) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (Scriptkiddi) [reply index.html:103]: fixed

Comment (SlothOfAnarchy) [reply index.html:103]: don't see it, but it's okay.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**